pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -63,7 +63,7 @@ func validateResources(resources []string, fldPath *field.Path) field.ErrorList
 	}
 
 	// x/*
-	resourcesWithWildcardSubresoures := sets.String{}
+	resourcesWithWildcardSubresources := sets.String{}
```

Comment:
looks like sets.String is deprecated: use generic Set instead.

Reply:
A bit outside the scope of a typos PR, but sure, fixed!